spec/array_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'recursive_inject'
+
+describe Array do
+  it "can add" do
+    expect([1,2,3].inject{|memory, element| memory + element}).to eq 6
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.